Resources/Prototypes/Entities/Clothing/OuterClothing/base_clothingouter.yml

```diff
@@ -68,6 +68,7 @@
         Heat: 0.90
         Radiation: 0.25
   - type: ToggleableClothing
+    slot: head
```

Comment:
oh, you can keep it only here

Reply:
You're missing that there are a few cloaks and an EVA softsuit that use toggleable clothing. I get your point regarding parental prototypes, but I think it should be everywhere for better understanding. I was even thinking about not including it in the parent hardsuit prototype, but decided to keep it for already existing custom prototypes to be operable.

Reply:
I have reconsidered your comment, you're right. Hardsuits can't really exist without helmets, so slot specification is kept in the parental prototype only.